submarine-workbench/workbench-web-ng/src/WEB-INF/web.xml

```diff
@@ -28,10 +28,34 @@
     <servlet-class>org.glassfish.jersey.servlet.ServletContainer</servlet-class>
     <init-param>
       <param-name>jersey.config.server.provider.packages</param-name>
-      <param-value>org.apache.submarine.server.rest</param-value>
+      <param-value>io.swagger.jaxrs.listing,org.apache.submarine.server.rest</param-value>
     </init-param>
     <load-on-startup>1</load-on-startup>
   </servlet>
+  <servlet>
+    <servlet-name>Jersey2Config</servlet-name>
+    <servlet-class>io.swagger.jersey.config.JerseyJaxrsConfig</servlet-class>
+    <init-param>
+      <param-name>api.version</param-name>
+      <param-value>1.0.0</param-value>
+    </init-param>
+    <init-param>
+      <param-name>swagger.api.basepath</param-name>
+      <param-value>http://localhost:8080/api/</param-value>
+    </init-param>
+    <load-on-startup>2</load-on-startup>
+  </servlet>
+  <servlet>
+    <servlet-name>Bootstrap</servlet-name>
+    <servlet-class>org.apache.submarine.server.Bootstrap</servlet-class>
+    <load-on-startup>2</load-on-startup>
+  </servlet>
+
+  <servlet-mapping>
+    <servlet-name>default</servlet-name>
+    <url-pattern>/v2/*</url-pattern>
```

Comment:
why use the `v2`?

Reply:
do you mean swagger v3?

Reply:
This path is for the user to check swagger.json? like `localhost:8088/v2/swagger.json`
it just an arbitrary name, no special meaning.
Maybe we could change to other name. Any suggestion?

Reply:
I think it should keep align with the version of submarine REST api, at now it used `v1`.